### ISSUE TITLE
My visual studio code i lagging

### ISSUE THREAD

**Issue by bielLacasa** [microsoft/vscode#215763]
Type: <b>Performance Issue</b>

I try to code with it, but my input has a bit of a delay, which makes it impossible to code properly.
The rest of the computer apps are working correctly, so I know it isnt the computer. I tried restarting but it didn't work.
I disabled all of the extensions to make sure it wasn't the problem. 

VS Code version: Code 1.90.1 (611f9bfce64f25108829dd295f54a6894e87339d, 2024-06-11T21:02:43.666Z)
OS version: Linux x64 6.5.0-35-generic snap
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz (8 x 1365)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|3, 4, 2|
|Memory (System)|15.41GB (12.02GB free)|
|Process Argv|--force-user-env --no-sandbox --use-gl=angle --use-angle=swiftshader --crash-reporter-id 2e8bc98d-401b-434e-a7ed-080a44f3bc2e|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|wayland|
</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
    0	   158	  4854	code main
    0	    32	  4856	   zygote
    0	    32	  4857	   zygote
    0	   268	  4934	window [1] (● main.cc - EXAMEN - Visual Studio Code)
    0	    95	  5229	window [2] (Issue Reporter)
    0	    63	  4891	   utility-network-service
    0	   110	  4968	extensionHost [1]
    0	    95	  4978	shared-process
    0	     0	  5241	     /bin/sh -c /usr/bin/ps -ax -o pid=,ppid=,pcpu=,pmem=,command=
    0	     0	  5242	       /usr/bin/ps -ax -o pid=,ppid=,pcpu=,pmem=,command=
    0	    79	  4979	fileWatcher [1]
    0	    79	  5013	ptyHost
    0	     0	  5035	     /usr/bin/bash --init-file /snap/code/161/usr/share/code/resources/app/out/vs/workbench/contrib/terminal/browser/media/shellIntegration-bash.sh
```

</details>
<details>
<summary>Workspace Info</summary>

```
|  Window (● main.cc - EXAMEN - Visual Studio Code)
|    Folder (EXAMEN): 81 files
|      File types: hpp(11) cor(9) inp(9) cpp(9) o(7) pdf(6) tar(5) cc(4)
|                  exe(4) e(3)
|      Conf files: makefile(5);
```

</details>
Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
jchc7451:31067544
chatpanelc:31048052
dsvsc021:30996838
724cj586:31013169
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
pythonprt:31056678
dwnewjupytercf:31046870

```

</details>

**Comment by shelvesdragon** [comment]
This is likely caused by the temporary fix in version 1.90.1 on the snap package that switched the OpenGL backend to software emulation in order to prevent the crashes reported with version 1.90.0 in #212494 . If it is related, your options for now are mentioned in this comment: https://github.com/microsoft/vscode/issues/212494#issuecomment-2168072242
Hope this helps!

**Comment by deepak1556** [comment]
Merging to https://github.com/microsoft/vscode/issues/212494, please follow the issue for updates.
